projects/plugins/jetpack/tests/php/modules/widgets/test_contact-info-widget.php

```diff
@@ -32,7 +32,7 @@ public function test_form_apikey_field_with_no_filter() {
 		$this->contact_info_widget->form( null );
 		$output_string = ob_get_clean();
 
-		$apikey_field_displayed = false === strpos( $output_string, '<input type="hidden" id="widget-widget_contact_info' );
+		$apikey_field_displayed = ! str_contains( $output_string, '<input type="hidden" id="widget-widget_contact_info' );
 		$this->assertTrue( $apikey_field_displayed );
```

Comment:
Looks like the three in this file could be switched to `assertStringContainsString` or `assertStringNotContainsString` too.

Reply:
👍 Done in 9f26940b3fa7a5f97869829ea041787461624558